contracts/adapters/swap/source/src/swap_source.rs

```diff
@@ -0,0 +1,62 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
```

Comment:
Is SwapSource open source / have a github repo that we can import the types from?

Reply:
Since we do not have any info releated to source types repo so we have not imported any types.